docs/api/web-contents.md

```diff
@@ -1306,9 +1306,11 @@ const requestId = webContents.findInPage('api')
 console.log(requestId)
 ```
 
-#### `contents.capturePage([rect])`
+#### `contents.capturePage([rect, stayHidden, stayAwake])`
 
 * `rect` [Rectangle](structures/rectangle.md) (optional) - The area of the page to be captured.
+* `stayHidden` boolean (optional) -  Keep the page hidden instead of visible.
+* `stayAwake` boolean (optional) -  Keep the system awake instead of allowing it to sleep.
```

Comment:
```suggestion
* `stayHidden` boolean (optional) -  Keep the page hidden instead of visible. Default is `false`.
* `stayAwake` boolean (optional) -  Keep the system awake instead of allowing it to sleep. Default is `false`.
```